src/components/inputField/InputField.stories.tsx

```
@@ -16,14 +16,20 @@ type Story = StoryObj<typeof InputField>;
+		return <InputField label={args.label}>{inputs}</InputField>;
 	},
 };
 
 export const Error: Story = {
 	args: {
 		label: "유효기간",
-		inputs: [<Input placeholder="MM" value="" onChange={() => {}} isError={true} />, <Input placeholder="YY" value="" onChange={() => {}} />],
 		errorMessage: <Description color={COLORS.ERROR}>숫자만 입력 가능합니다.</Description>,
 	},
+	render: (args) => {
+		const inputs = [<Input placeholder="MM" value="" onChange={() => {}} isError={true} />, <Input placeholder="YY" value="" onChange={() => {}} />];
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)